### PR TITLE
Table 1 update

### PR DESCRIPTION
**Pull request by josephignace**
Table 1 script includes covariates listed in the protocol (combines table1 and extendedtable1). 
extendedtable1 script removed.
plot labels updated for age models

### CODE REVIEW

File: analysis/descriptives/table1.R
[the remark this answers is not in the document]
Comment (josephignace) [reply analysis/descriptives/table1.R:1]: Fixed.

File: analysis/descriptives/table1.R [hunk 1]
```diff
@@ -188,8 +252,20 @@ df$subcharacteristic <- factor(df$subcharacteristic,
                                           "West Midlands",
                                           "Yorkshire/Humber",
                                           "Care home resident",
-                                          "Missing"))
-
+                                          "0",
+                                          "1-5",
+                                          "6+",
+                                          "Healthcare worker",
+                                          "TRUE", "FALSE", "TRUE", "FALSE",
```
[the remark this answers is not in the document]
Comment (josephignace) [reply analysis/descriptives/table1.R:259]: Removed extra words.

File: analysis/descriptives/table1.R [hunk 2]
```diff
@@ -152,18 +207,27 @@ df$subcharacteristic <- factor(df$subcharacteristic,
                                           "West Midlands",
                                           "Yorkshire and The Humber",
                                           "Care home resident",
+                                          "0",
+                                          "1-5",
+                                          "6+",
+                                          "Healthcare worker",
+                                          "TRUE", "FALSE", "TRUE", "FALSE",
```
[the remark this answers is not in the document]
Comment (josephignace) [reply analysis/descriptives/table1.R:214]: Removed extra words.

File: lib/plot_labels.csv [hunk 3]
```diff
@@ -15,9 +15,9 @@ unvax,Unvaccinated (Jun 1 2021 - Dec 14 2021)
 main,All COVID-19
 day0,Day 0
 sub_age_18_39,Age group: 18-39
-sub_age_40_59,Age group: 40-59
-sub_age_60_79,Age group: 60-79
-sub_age_80_110,Age group: 80-110
+sub_age_40_64,Age group: 40-64
+sub_age_65_84,Age group: 65-84
+sub_age_85_110,Age group: 85+
```
[the remark this answers is not in the document]
Comment (josephignace) [reply lib/plot_labels.csv:20]: Keep old age format.

[the remark this answers is not in the document]
Comment (venexia) [reply lib/plot_labels.csv:20]: This has been fixed in the code but not `plot_labels.csv`.

**Review by venexia — APPROVED** [review]
Hi @josephignace. This looks good - thank you. Just one v small change to `plot_labels.csv` for consistency.

**Pull request merged**